corona-warn-app/cwa-app-android
# Certificate expired adjustments (EXPOSUREAPP-8837)

This PR addresses Exposureapp-8837. (see ticket for details & screenshots)

The current check (certificate is expired) runs every 24h (worker).
This PR adds an additonal check for Vaccination/Test-Certifcates if opened by the user.
(A more advanced solution might come in the next sprint)

How to test:
-Scan a Vaccination Cert or Recovery Cert which will expire soon
-Check if the correct expiration text is shown after opening the cert view.

Screenshot:
![image](https://user-images.githubusercontent.com/46747780/128997483-da809035-7211-4a8a-b55f-de227ab00559.png)
